src/node_modules.cc

```diff
@@ -579,6 +579,86 @@ void GetCompileCacheEntry(const FunctionCallbackInfo<Value>& args) {
       isolate, v8::Null(isolate), names.data(), values.data(), names.size()));
 }
 
+static void InitImportMetaLazyGetter(
+    Local<v8::Name> name, const v8::PropertyCallbackInfo<Value>& info) {
```

Comment:
Nit: remove the `v8::` qualifier and add `v8::PropertyCallbackInfo` to the using section at the top if it is not already there.